apex/helper/system_activation_helper.go

```diff
@@ -144,12 +156,26 @@ func ActivateSystemClientSystem(ctx context.Context, clientAPI *apexClient.APICl
 // @return string current activation token. If it is an empty string we were not able to extract and user should try to post a new one
 func getApexActivationToken(ctx context.Context, clientAPI *apexClient.APIClient, systemID string, storageType apexClient.StorageProductEnum) string {
 	post := clientAPI.StorageSystemTokensAPI.StorageSystemTokensCreate(ctx)
+	accessToken := "test-dummy-token"
+
+	if storageType == apexClient.STORAGEPRODUCTENUM_POWERSCALE {
+		accessToken, _ = GetNewToken(ctx)
```

Comment:
This is for the Apex DI token, this code should not be used as part of the system_activation code.

Reply:
Talked in slack